src/aat/resources/features/F-1008 - Submit Event Creation Handle Case Links/F-1008-VerifyCaseLinks.td.json

```diff
@@ -0,0 +1,45 @@
+{
+	"title": "should retrieve case links for a specified case reference",
+	"_guid_": "F-1008-VerifyCaseLinks",
+
+	"productName": "CCD Data Store",
+	"operationName": "Retrieve case links for an existing case reference",
+
+	"method": "GET",
+	"uri": "/testing-support/case-link/{cid}",
+
+	"specs": [
+		"to verify that the Case Link has been created in the CASE_LINK table with correct value"
+	],
+
+  "users": {
+    "invokingUser": {
+      "_extends_": "BeftaMasterCaseworker"
+    }
+  },
+
+	"request": {
+		"headers": {
+			"_extends_": "Common_Request_Headers",
+      "experimental": "true"
+		},
+		"pathVariables": {
+			"cid": "${[scenarioContext][parentContext][childContexts][F-1008_CreateCasePreRequisiteCaseworkerBase][testData][actualResponse][body][id]}"
+		}
+	},
+
+	"expectedResponse": {
+		"_extends_": "Common_200_Response",
+		"headers": {
+			"Content-Encoding": "gzip",
+			"Content-Length": "[[ANYTHING_PRESENT]]"
+		},
+		"body": {
+      "CaseLinks" : [ {
+        "CaseId" : "[[ANYTHING_PRESENT]]",
+        "LinkedCaseId" : "[[ANYTHING_PRESENT]]",
```

Comment:
Shouldn;t these be updated to reflect the expected values?

Reply:
Done [2e98c6c](https://github.com/hmcts/ccd-data-store-api/pull/1719/commits/2e98c6c69d70f77077a6395aad95e21a9135f99a)